UnitedIncome/components
# Slider tooltip sticky fix

### Description
Fix slider when tooltip sticky variant is given but not tooltip sticky position

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.
1. Go to the slider story
1. Make sure there's a tooltip sticky variant provided but not a tooltip sticky position
1. Make sure the 0 is not in green

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.
